samples/datatype-sample/README.md

```diff
@@ -10,19 +10,94 @@ products:
 
 # Different data input sample for Kubernetes Marketplace Offer
 
-This sample demonstrates how to override array objects inside helm chart for Kubernetes Marketplace Offer
+This sample demonstrates 
```

Comment:
Do you want to include updated values.yaml and deployments.yaml in this sample folder?

Reply:
Those values are already in the main azure-vote. The link is included at the bottom of the page in the Note section